Realm.xcodeproj/xcshareddata/xcschemes/OSX.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0600"
+   LastUpgradeVersion = "0610"
```

Comment:
It's used for the thing where Xcode nags you to update your project settings to avoid doing it when you've explicitly switched away from the new default. Should be no need to ever care about it.